src/schema/generated-schema-types.ts

```diff
@@ -977,6 +977,26 @@ export interface NameId {
 	name?: string | null;
 }
 
+export interface AuthResponse {
+	authToken?: AuthToken[] | null;
```

Comment:
usually auth token is stored as cookie only, we should not store it somewhere else

Reply:
I think you are right and this is actually already a problem, see https://github.com/apollographql/apollo-client/issues/3592.

It looks like we should likely be using `fetchPolicy: 'no-cache'` for the `AuthRequest` endpoint to workaround the issue.

Reply:
Thanks @silentsakky and @pl12133.  I agree we should not be storing/caching the auth tokens here.

Reply:
or maybe if authtoken is not used anywhere in the application then remove it from schema? that way we will not be storing in cache at all